main_test.go

```diff
@@ -383,6 +383,53 @@ func TestPublishersEndpoints(t *testing.T) {
 				assert.Equal(t, "wrong cursor format in page[after] or page[before]", response["detail"])
 			},
 		},
+		{
+			description: `GET with "from" query param`,
+			query:       "GET /v1/publishers?from=2018-11-26T00:56:23Z",
```

Comment:
```suggestion
			query:       "GET /v1/publishers?from=2018-11-10T00:56:23Z",
```

I'd use this query...